src/docfx/validation/ContentValidator.cs

```diff
@@ -67,6 +68,7 @@ public void ValidateLink(FilePath file, SourceInfo<string> link, MarkdownObject
                     Monikers = origin.GetZoneLevelMonikers(),
                     ZonePivots = origin.GetZonePivots(),
                     TabbedConceptualHeader = origin.GetTabId(),
+                    IsInHtmlBlock = isInHtml,
```

Comment:
Instead of saying `isInHtml`, consider make `LinkText` nullable and use `null` to represent _"can I reliably get the link text for this link?"_. In addition to be an `HtmlInline`, there could be other reasons in the future where we cannot reliably retrieve the link text.

Reply:
isInHtml is simply used to indicate whether the link is in Html, not about whether text present.

Reply:
consider turn this into an enum:

```
LinkType {
    Default
    Autolink
    Html
    Icon
}
```